src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return color.toString();
     }
```

Comment:
You're creating a new instance of Random every time the method is called, which is unnecessary. Consider reusing a single Random instance. Also, instead of using `toString()` for getting the name of the enum constant, use `name()` as per the checklist.